### PR TITLE
Clustergrammer build

### PR DESCRIPTION
**Pull request by JamesJeffryes**
So unfortunately I was editing the wrong file earlier. This version should actually enable the widget in the built image

### CODE REVIEW

File: scripts/install_narrative_docker.sh [hunk 1]
```diff
@@ -110,5 +114,6 @@ jupyter nbextension install $(pwd)/codeCell --symlink --sys-prefix
 jupyter nbextension enable codeCell/main --sys-prefix
 
 jupyter nbextension enable --py --sys-prefix widgetsnbextension
+jupyter nbextension enable --py --sys-prefix clustergrammer_widget
```
Comment (JamesJeffryes) [inline scripts/install_narrative_docker.sh]: It might make sense to replace this logic with a call to `nbextention/install.sh`

**Comment by JamesJeffryes** [comment]
At this point I wonder if it might almost be easier to update the narrative base image but I don't know how to do that.

**Comment by briehl** [comment]
I was thinking that might be the way to go, too. I don't think that's written down anywhere, either, since the most recent update.

I'll dredge my memory for how to do it, document it, and make the update. I think having this setup for running it locally is fine, too.

**Comment by JamesJeffryes** [comment]
@briehl Should we merge this or wait till the base image is updated? I'd kinda like to get this working for demos Fri

**Comment by briehl** [comment]
Let's merge now and see if it works. I can work on the base image in the meantime.

**Pull request merged**